platforms/core-configuration/declarative-dsl-provider/src/main/kotlin/org/gradle/internal/declarativedsl/project/dependencyConfigurationSchema.kt

```diff
@@ -185,14 +185,17 @@ class ImplicitDependencyCollectorFunctionResolver(configurations: DependencyConf
 
     private
     fun getDependencyCollectorGetter(receiverClass: KClass<*>, configurationName: String): KFunction<*>? = receiverClass.functions
```

Comment:
This method still operates on configuration names. Does this also need to be updated?